common/src/main/kotlin/no/nav/su/se/bakover/common/Config.kt

```diff
@@ -23,6 +23,7 @@ object Config {
     val azureGroupSaksbehandler = env["AZURE_GROUP_SAKSBEHANDLER"] ?: "0ba009c4-d148-4a51-b501-4b1cf906889d"
     val azureGroupVeileder = env["AZURE_GROUP_VEILEDER"] ?: "062d4814-8538-4f3a-bcb9-32821af7909a"
     val suInntektAzureClientId = env["SU_INNTEKT_AZURE_CLIENT_ID"] ?: "9cd61904-33ad-40e8-9cc8-19e4dab588c5"
+    val oppgaveClientId = env["SU_OPPGAVE_CLIENT_ID"] ?: "41ca50ba-e44f-4bc8-9e31-b745a0041926"
```

Comment:
Tror de som har SU_ prefixen er våre ting, så kan nok sløyfe den her?

Reply:
Hva brukes den til? Kan den variere per miljø? Skader det at den ligger her? :D

Reply:
Vi hadde vel et eget su-oppgave repo som denne refererer til. Tenkte bare å bytte navn til OPPGAVE_CLIENT_ID. Det er azure id-en til Oppgave, som vi bruker for å få onBehalfOf token